roles/swift-account/handlers/main.yml

```diff
@@ -4,8 +4,8 @@
     name: "{{ item }}"
     state: restarted
   with_items:
-    - swift-account
-    - swift-account-auditor
-    - swift-account-reaper
-    - swift-account-replicator
+    - "{{ (ansible_os_family == 'RedHat') | ternary('openstack-', '') }}swift-account"
```

Comment:
We need to switch to openstack-meta. Define service names there. Also we write override systemd as most rhosp openstack systemd were pulling in config fragments from several places.